examples/disable-persistence-queue-traces/rendered_manifests/configmap-agent.yaml

```diff
@@ -159,8 +159,12 @@ data:
       k8sattributes/metrics:
         extract:
           annotations:
+          - from: namespace
+            key: splunk.com/metricsSourceType
+            tag_name: com.splunk.sourcetype
```

Comment:
I think that we are extracting sourcetypes from a pod only, if we want to change that then change it in logs pipeline/documentation too.

Reply:
right, removed